.pre-commit-config.yaml

```diff
@@ -1,12 +1,12 @@
 repos:
-  - repo: git://github.com/dnephin/pre-commit-golang
+  - repo: https://github.com/dnephin/pre-commit-golang
     rev: master
     hooks:
       - id: go-fmt
```

Comment:
Does golangci-lint auto fix issues, if yes, I guess this is not needed as we have go-fumpt defined there?